src/control/cmd/daos/flags_test.go

```diff
@@ -282,3 +282,49 @@ func TestFlags_ConsModeFlag(t *testing.T) {
 		})
 	}
 }
+
+func TestFlags_ContTypeFlag(t *testing.T) {
+	for name, tc := range map[string]struct {
+		arg       string
+		expFlag   *ContTypeFlag
+		expString string
+		expErr    error
+	}{
+		"unset": {
+			expErr: errors.New("empty container type"),
+		},
+		"invalid": {
+			arg:    "snausages",
+			expErr: errors.New("unknown container type"),
+		},
+		"valid": {
+			arg: "POSIX",
```

Comment:
It would be good to add a test case to prove it's case-insensitive

Reply:
D'oh. I'd meant to update this one with mixed case. I'll fix that.